pybikes/data/bysykkel_graphql.json

```diff
@@ -0,0 +1,18 @@
+{
+    "instances":[
+        {
+            "tag":"trondheim-bysykkel",
+            "meta":{
+                "latitude":63.4307238,
+                "country":"NO",
+                "name":"Bysykkel",
+                "longitude":10.3936781,
+                "city":"Trondheim",
+                "city_code":"TRD",
+                "systemId": "trondheim"
```

Comment:
If you move `systemId` outside of `meta`, it will be available on initialization, see: [pybikes/nextbike.py](https://github.com/eskerda/pybikes/blob/master/pybikes/nextbike.py#L31) and [pybikes/data/nextbike.json](https://github.com/eskerda/pybikes/blob/master/pybikes/data/nextbike.json#L4-L12).